### PR TITLE
added MR2mini buttons support template

### PR DESCRIPTION
**Pull request by RLOVD**
Шаблоны mrm2mini с поддержкой маппинг - матриц и маппинг матриц кнопок

### CODE REVIEW

File: wb-mqtt-serial-templates/config-wb-mrm2-mini-fw1_7.json [hunk 1]
```diff
@@ -0,0 +1,1591 @@
...
+                                "set",
+                                "toggle"
+                            ]
+                        },
+            
+                        "in1_dp": {
+                            "group": "push_bt_in1_2",
+                            "title": "dp_title",
+                            "order": 3,
+                            "address": 680,
```
Comment (ExplorerOL) [inline wb-mqtt-serial-templates/config-wb-mrm2-mini-fw1_7.json:1404]: При конфигурировании работы действий по кнопочной маппинг матрице от Input2 получаю ошибки:
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: INFO: [modbus] Init: Input 1 Input mode: setup register <modbus:81:holding: 9> <-- 6 (0x6)
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: WARNING: [modbus] failed to write: <modbus:81:holding: 680>: Serial protocol error: illegal data value
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: INFO: [modbus] Init: Input 2 dp_time_title: setup register <modbus:81:holding: 1141> <-- 1000 (0x3e8)
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: INFO: [modbus] Init: CONF_A3_A4: setup register <modbus_io-tcp:13:1:holding: 19505> <-- 0x0 (0x0)
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: WARNING: [modbus] failed to write: <modbus:81:holding: 616>: Serial protocol error: illegal data value
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: INFO: [modbus] Init: Input 2 lp_hold_time_title: setup register <modbus:81:holding: 1101> <-- 500 (0x1f4)
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: WARNING: [modbus] failed to write: <modbus:81:holding: 744>: Serial protocol error: illegal data value
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: WARNING: [modbus] failed to write: <modbus:81:holding: 552>: Serial protocol error: illegal data value
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: WARNING: [modbus] failed to write: <modbus:81:holding: 681>: Serial protocol error: illegal data value
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: WARNING: [modbus] failed to write: <modbus:81:holding: 617>: Serial protocol error: illegal data value
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: WARNING: [modbus] failed to write: <modbus:81:holding: 745>: Serial protocol error: illegal data value
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: INFO: [modbus] Init: CONF_A5_A6: setup register <modbus_io-tcp:13:1:holding: 19510> <-- 0x0 (0x0)
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: WARNING: [modbus] failed to write: <modbus:81:holding: 553>: Serial protocol error: illegal data value
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: INFO: [modbus] Init: Input 2 Input mode: setup register <modbus:81:holding: 10> <-- 6 (0x6)
Feb 03 17:05:13 wirenboard-ATHXPBSP wb-mqtt-serial[29819]: INFO: [modbus] Init: CONF_A7_A8: setup register <modbus_io-tcp:13:1:holding: 19511> <-- 0x0 (0x0)

Похоже часть регистров осталась от MR6: 680, 616, 744, 552, 681,  617, 745, 553

File: wb-mqtt-serial-templates/config-wb-mrm2-mini-fw1_7.json [hunk 2]
```diff
@@ -0,0 +1,1591 @@
...
+                            "min": 0,
+                            "max": 100,
+                            "default": 50,
+                            "order": 1
+                        },
+                        "out1_actions": {
+                            "group": "out_actions_2",
+                            "title": "Output 1",
+                            "order": 2,
+                            "address": 392,
```
Comment (ExplorerOL) [inline wb-mqtt-serial-templates/config-wb-mrm2-mini-fw1_7.json:1157]: Регистры 392, 393 тоже надо исправить

**Review by ExplorerOL — CHANGES_REQUESTED** [review]
Оставил комментарии

**Review by ExplorerOL — APPROVED** [review]
Больше у меня нет замечаний

**Comment by pgasheev** [comment]
Шаблон добавлен в #433, более не актуально

**Pull request closed without merging**